automatic/vscode-insiders.install/README.md

```diff
@@ -0,0 +1,27 @@
+# <img src="https://cdn.jsdelivr.net/gh/felipecassiors/chocolatey-coreteampackages@master/icons/vscode-insiders.png" width="48" height="48"/> [vscode.install](https://chocolatey.org/packages/vscode)
```

Comment:
you are pointing to the `vscode` package here, please change it to point to the `vscode-insider.install` package. (The link text will also need to be changed)